jsk_perception/sample/sample_darknet_yolo_object_detection.launch

```diff
@@ -0,0 +1,25 @@
+<launch>
```

Comment:
Do you mean the sample_darknet_yolo_object_detection.launch is not the sample for `darknet_object_detection` node, but for `darknet_yolo_object_detection.launch`?

Could you please describe why you don't like to use `sample_darknet_object_detection.launch` in detail?